test/controllers/declaration/DucrEntryControllerSpec.scala

```diff
@@ -128,6 +147,21 @@ class DucrEntryControllerSpec extends ControllerSpec with GivenWhenThen {
       }
     }
 
+    onEveryDeclarationJourney(withStatus(DeclarationStatus.AMENDMENT_DRAFT)) { implicit request =>
+      "return 303 (SEE_OTHER) and redirect" which {
+        "redirects to /saved-summary" in {
```

Comment:
Need to specify that this is testing the situation where the draft dec is an amendment draft